tests/javascript/captionjs/spec/javascripts/fixtures/caption-fixture.html

```diff
@@ -0,0 +1,20 @@
+<div class="single">
+    <img src="base/images/joomla_black.png" class="test" title="Joomla Title 1" />
+</div>
+<hr/>
+<div class="multiple">
```

Comment:
As stated above, any reason using classes for the divs? As they are unique you can also use ids?